adm_program/system/classes/TablePhotos.php

```diff
@@ -220,7 +220,40 @@ public function isVisible()
             return false;
         }
 
-        return true;
+        if($gCurrentUser->isAdministrator()) {
+            error_log("[admin - allow access]");
+            return true;
+        }
+        //echo "pho_id:[".$this->getValue('pho_id')."]";
+        $sql = 'SELECT * 
+                      FROM '.TBL_DATES.'
+                     WHERE dat_photo_id = '.$this->getValue('pho_id');
+        $eventsTiedToPhotoAlbums = $this->db->query($sql)->fetchAll();
+        if(!sizeof($eventsTiedToPhotoAlbums)) {
```

Comment:
Better will be to work with `$eventsTiedToPhotoAlbums->rowCount() === 0` because you want to check this.

Reply:
replaced sizeof with rowCount